setup.py

```diff
@@ -22,7 +17,8 @@
 os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
```

Comment:
Maybe it can be simplified with `pathlib`? I think they are equivalent 
```suggestion
from pathlib import Path
os.chdir(Path(__file__).parent.absolute())))
```

Reply:
Oh sorry I didn't see it was not part of your changes :sweat_smile: